### PR TITLE
Backport PR #4189: correct documentation

### PR DESCRIPTION
**Pull request by registerrier**
Correct documentation

**Description**

This pull request backports #4189 

**Dear reviewer**

### CODE REVIEW

File: docs/user-guide/references.rst [hunk 1]
```diff
@@ -17,7 +17,7 @@ Glossary
     1D Analysis
       1D analysis or spectral analysis where data are reduced to a simple 1D
       geometry along the reconstructed energy axis. In Cherenkov astronomy,
-      this is classically performed with an OFF background measurement [Piron2001]_.
+      this is classically performed with an OFF background measurement, see WStat.
```
Comment (bkhelifi) [inline docs/user-guide/references.rst:20]: Why not making a reference to wstat in fit-statistics?

File: examples/tutorials/starting/analysis_2.py [hunk 2]
```diff
@@ -36,7 +36,7 @@
 
 We can then proceed with data reduction with a loop over all selected
 observations to produce datasets in the relevant geometry and stack them
-together (i.e. sum them all).
+together (i.e.sum them all).
```
Comment (bkhelifi) [inline examples/tutorials/starting/analysis_2.py:39]: The initial text was correct, is not it? Why removing this space....?

File: examples/tutorials/starting/overview.py [hunk 3]
```diff
@@ -481,7 +483,7 @@
 #
 # -  To learn more about those classes, go to the API docs (links are in
 #    the introduction at the top).
-# -  To learn more about other parts of Gammapy (e.g. Fermi-LAT and TeV
+# -  To learn more about other parts of Gammapy (e.g.Fermi-LAT and TeV
```
Comment (bkhelifi) [inline examples/tutorials/starting/overview.py:486]: The initial comment was correct, with a space...

**Review by bkhelifi — COMMENTED** [review]
Thanks a lot with this great imrovement. I have some minor comments...

File: examples/tutorials/starting/overview.py (hunk 3 above)
Comment (registerrier) [reply examples/tutorials/starting/overview.py:486]: There was a weird character (not visible here) which I removed but forgot to put the space back in that case. I will correct.

**Pull request merged**